### ISSUE TITLE
Ubuntu Studio backup problems

### ISSUE THREAD

**Issue by Zekodon** [chamuco/respin#22]
I made a backup of a build of ubuntu studio in a VM. I copied the output .iso file and tried to boot it into a different virtual machine. I was able to boot into GRUB and first I tried to 'try live system' option and it was hung. Then I restarted the vm and when I got to grub I tried the 'just install the thing' option. Still nothing. After that I tried the 'start in graphic compatibility mode' option and the screen changed: I got a black screen followed by the ubuntu studio loading screen for about 30 minutes and then.... BLACK SCREEN. I don't know what I did wrong, ubuntu doesn't use mdm or anything other than what the ubuntu base offers.

DL;DR it didn't work, can anyone help me out?

**Comment by chamuco** [comment]
@Zekodon - Can you please try installing from the launchpad ppa? That code is most up to date. You can install with the below commands:

sudo apt-add-repository ppa:sergiomejia666/respin

sudo apt-get update

sudo apt-get install respin -y

Please let me know if you still experience the same issue

**Comment by Zekodon** [comment]
Okay, I wasn't able to install it because I can't get the two dependencies 'isolinux' and 'syslinux' . I also uninstalled respin and respin-gtk from the original repo i had installed it from. Any help?

**Comment by chamuco** [comment]
@Zekodon - Can you download those packages separately with 'sudo apt-get
install isolinux -y' and 'sudo apt-get install syslinux-utils -y' ? Can you
please provide the output of 'sudo cat /etc/*release' ?

**Comment by Zekodon** [comment]
So my output for the 'sudo cat /etc/*release' went like this:

studio@studio-VirtualBox:~$ sudo cat /etc/*release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.4 LTS"
NAME="Ubuntu"
VERSION="14.04.4 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.4 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"

When I tried to install isolinux: 

studio@studio-VirtualBox:~$ sudo apt-get install isolinux -y
Reading package lists... Done
Building dependency tree  
Reading state information... Done
Package isolinux is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'isolinux' has no installation candidate

When I tried to install syslinux-utils

studio@studio-VirtualBox:~$ sudo apt-get install syslinux-utils -y
Reading package lists... Done
Building dependency tree  
Reading state information... Done
Package syslinux-utils is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'syslinux-utils' has no installation candidate

TL;DR There isn't an installation candidate.

**Comment by chamuco** [comment]
@Zekodon - Thank you for providing that info. I am going to test this on
Ubuntu Studio 14.04 and will check what is causing the installation issues
with the packages isolinux and sysutils.

**Comment by chamuco** [comment]
@Zekodon - I updated the code to remove the unnecessary dependencies but
there is an issue with the backup option as this time. The dist option is
working at this time. Please uninstall and reinstall either from the Ubuntu
deb file or from the ppa.

**Comment by Zekodon** [comment]
I uninstalled it, then reinstalled it from the Ubuntu deb file. After I made The ISO file and ran it in a VM. I got the same Result as before:
1. Boots into grub.
2. Selects 'live' install.
3. Screen doesn't change at all.

Or
1. Boots into grub.
2. Selects 'install right now' button.
3. Screen stays the same for a while. Then ... 
4. Black screen forever.

**Comment by chamuco** [comment]
@Zekodon - Are you using version 2.0.6? If you run 'sudo respin' without an
argument it will display the version number. If you are running an older
version you would need to run 'sudo apt-get update' followed by 'apt-get
install respin -y'.
